R/getMetadataComment.R

```diff
@@ -0,0 +1,23 @@
+#' getMetadataComment
+#'
+#' Helper function extract a metadata comment
+#'
+#' @md
+#' @param x object the metadata should be extracted from
+#' @param name name of the metadata to be extracted (e.g. unit)
+#' @author Jan Philipp Dietrich
+#' @examples
+#' x <- as.magpie(1)
+#' getComment(x) <- c(" description: example description", " unit: kg")
+#' getMetadataComment(x, "unit")
+#' getMetadataComment(x, "description")
+#' @export
+
+getMetadataComment <- function(x, name) {
```

Comment:
```suggestion
getMetadata <- function(x, name) {
```
I find "Comment" a bit misleading, it reads to me like you can extract the "comment" from the metadata. It is actually `getMetadataFromComment`, right? Maybe to keep it short just `getMetadata`?